src/guides/arguments.md

```diff
@@ -0,0 +1,31 @@
+## Argument Flags :
+Function argument flags mark how arguments within a function may be used (or hence, not used at all), it will be displayed near or bleow the function argument as example :
+(- `userID` `(Flag : Required)`) etc
```

Comment:
```suggestion
## Argument Flags
Function argument flags mark how arguments within a function may be used (or hence, not used at all), it will be displayed near or below the function argument.
Example: (- `userID` `(Flag: Required)`) etc.
```